packages/event-target/redis-event-target/src/index.ts

```diff
@@ -5,13 +5,21 @@ import { CustomEvent } from '@whatwg-node/events';
 export type CreateRedisEventTargetArgs = {
   publishClient: Redis | Cluster;
   subscribeClient: Redis | Cluster;
+  serializer?: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    stringify: (message: any) => string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    parse: (message: string) => any;
```

Comment:
Do you think, it might make sense to use `unknown` instead of `any` here?

Reply:
:heavy_check_mark: